db/schema.rb

```diff
@@ -816,6 +816,7 @@
     t.string "aasm_state", default: "received"
     t.string "hostname"
     t.string "tags", default: [], array: true
+    t.integer "spam_score", default: 0
```

Comment:
How do we know integer will be the right type for this?

Reply:
a demo i saw returned an integer.

we don't.
but its a place to start and can be changed.